docs/configuration-reference/platforms/aws.md

```diff
@@ -178,7 +178,7 @@ worker_pool "my-worker-pool" {
 | `os_version`                  | Flatcar Container Linux version to install. Version such as "2303.3.1" or "current".                                                                                                       | "current"       | false    |
 | `dns_zone`                    | Route 53 DNS Zone.                                                                                                                                                                         | -               | true     |
 | `dns_zone_id`                 | Route 53 DNS Zone ID.                                                                                                                                                                      | -               | true     |
-| `ssh_pubkeys`                 | SSH public keys for user `core`.                                                                                                                                                           | -               | true     |
+| `ssh_pubkeys`                 | SSH public keys for user `core`. This is the full public key as shown by `ssh-add -L`.                                                                                                     | -               | true     |
```

Comment:
This assumes the user users ssh-agent. I sometimes encounter tech folks who don't know about ssh-agent or wouldn't use it for some other reason. Also, assuming most users will have the key pair as local files, I would consider using the word "contents" to make it very clear they have to paste the contents of the public key file rather than a reference to the file path.

Reply:
Updated.